### ISSUE TITLE
[NemaGFX] Spinner animation rotates strangley

### ISSUE THREAD

**Issue by KlemenDEV** [kisvegabor/lvgl#7276]
### LVGL version

current master branch

### What happened?

We use code from https://docs.lvgl.io/master/widgets/spinner.html to define the spinner, a completely default implementation.

The animation looks like this when LV_USE_NEMA_GFX is 1, no OS enabled:

![bug](https://github.com/user-attachments/assets/ea3a205d-b73e-41a7-b5d4-85c78c3d6871)

**Comment by liamHowatt** [comment]
Thanks, will take a look soon!

**Comment by KlemenDEV** [comment]
Perfect, if you need any more info, let me know.

I am testing master branch on STM32U5G9 with DSI and LTDC

**Comment by KlemenDEV** [comment]
I know this is unrelated, but I figured this may be a place to ask since there is no "Discussions" section on this repo.

https://docs.lvgl.io/master/details/integration/renderers/nema_gfx.html says OS support is experimental. What part should I be careful of, since I tried it and it seemed to work at first glance (using LV_OS_CMSIS_RTOS2).

But still this bug reported in this ticket happens with and without OS
